astro/src/content/blog/sharing-custom-oauth-claims-with-a-asp-netcore-app.mdx

```diff
@@ -1,7 +1,8 @@
 ---
 publish_date: 2020-05-27 
 title: ASP.NET Core Example with  Custom Claims
-description: We'll configure FusionAuth and extend the ASP.NET Core web application to display role and custom claims from a JWT.
+seoTitle: "ASP.NET Core Example: Display Custom Claims from a JWT"
+description: See a working ASP.NET Core example that adds custom JWT claims using FusionAuth. Extend ASP.NET Identity with role-based or app-specific data.
```

Comment:
```suggestion
description: See a working ASP.NET Core example that adds custom JWT claims using FusionAuth. Allow your ASP.NET users' identity to be extended with role-based or app-specific data.
```

Reply:
asp.net Identity is a specific thing: https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-9.0&tabs=visual-studio